examples/twobee/twobee/app.py

```diff
@@ -0,0 +1,38 @@
+import random
+import toga
+from toga.style.pack import Pack, COLUMN, CENTER
+
+HEADS = toga.Image('https://beeware.org/static/images/brutus-270.png')
+TAILS = toga.Image('https://beeware.org/project/projects/libraries/toga/toga.png')
+TOSS = toga.Image('https://beeware.org/project/projects/tools/cricket/cricket.png')
+IMG_SIZE = toga.style.Pack(height=75, width=75)  # TODO be nice if didn't have to specify size
```

Comment:
This should be possible - you just have to allocate how the images will spread their width over the available space in the row.